testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java

```diff
@@ -161,77 +161,88 @@
         )),
         WILDFLY_17_0("WildFly17.0", WILDFLY_16_0),
         WILDFLY_18_0("WildFly18.0", WILDFLY_17_0),
-
-        EAP62("EAP62", Arrays.asList(
-                "org.jboss.as.appclient",
-                "org.jboss.as.clustering.infinispan",
-                "org.jboss.as.clustering.jgroups",
-                "org.jboss.as.cmp",
-                "org.jboss.as.configadmin",
-                "org.jboss.as.connector",
-                "org.jboss.as.deployment-scanner",
-                "org.jboss.as.ee",
-                "org.jboss.as.ejb3",
-                "org.jboss.as.jacorb",
-                "org.jboss.as.jaxr",
-                "org.jboss.as.jaxrs",
-                "org.jboss.as.jdr",
-                "org.jboss.as.jmx",
-                "org.jboss.as.jpa",
-                "org.jboss.as.jsf",
-                "org.jboss.as.jsr77",
-                "org.jboss.as.logging",
-                "org.jboss.as.mail",
-                "org.jboss.as.messaging",
-                "org.jboss.as.modcluster",
-                "org.jboss.as.naming",
-                "org.jboss.as.pojo",
-                "org.jboss.as.remoting",
-                "org.jboss.as.sar",
-                "org.jboss.as.security",
-                "org.jboss.as.threads",
-                "org.jboss.as.transactions",
-                "org.jboss.as.web",
-                "org.jboss.as.webservices",
-                "org.jboss.as.weld",
-                "org.jboss.as.xts",
-                // This module was added in EAP70, but we move it to the EAP62 extension list to allow the test passing
-                // without adding it to the host-exclude section. We don't want to expose it in the host-exclude.
-                "org.wildfly.extension.mod_cluster"
+        WILDFLY_19_0("WildFly19.0", WILDFLY_18_0,Arrays.asList(
```

Comment:
I'm not sure why this was not spotted before but once I added the `org.wildfly.extension.metrics` to more recent WIldFly version, this was spotted in the CI